exporter/opentelemetry-exporter-zipkin/test-requirements.txt

```diff
@@ -0,0 +1,22 @@
+asgiref==3.7.2
+attrs==23.2.0
+Deprecated==1.2.14
+flaky==3.7.0
+importlib-metadata==6.11.0
+iniconfig==2.0.0
+packaging==23.2
+pluggy==1.4.0
+py==1.11.0
+py-cpuinfo==9.0.0
+pytest==7.1.3
+pytest-benchmark==4.0.0
+tomli==2.0.1
+typing_extensions==4.10.0
+wrapt==1.16.0
+zipp==3.17.0
+-e opentelemetry-api
+-e exporter/opentelemetry-exporter-zipkin-json
+-e exporter/opentelemetry-exporter-zipkin-proto-http
+-e opentelemetry-sdk
+-e opentelemetry-semantic-conventions
```

Comment:
Nit: Do we need to install these in order to not produce warning messages?

Reply:
what warning messages?

Reply:
Like if we editable install `opentelemetry-sdk` that takes a dependency on `opentelemetry-semantic-conventions` without it existing, does pip not output a warning message that it is not currently installed or no?

Reply:
Not really, these are added because they showed up in the resulting `pip freeze` after running `tox -e py38` command for this exporter. We may have unnecessary requirements in this file, that happens because of how we have configured `tox` [here](https://github.com/open-telemetry/opentelemetry-python/blob/d5f4946ef0b0c0480edd56bfb21052a2246efe73/tox.ini#L110). I want to refactor this in a subsequent PR so that we only have the exact dependencies.